src/App.css

```diff
@@ -1,38 +1,79 @@
-.App {
-  text-align: center;
+.calculatorContainer {
+    width: fit-content;
+    background-color: rgb(3, 3, 90);
+    border-radius: 0.75rem;
+    border: solid 7px rgb(3, 3, 90);
+    margin: 50%;
```

Comment:
you could try maring: auto maybe?
Also do you only want left and right margin for this one/ or up and down as well?